src/renderers/shared/fiber/isomorphic/ReactCoroutine.js

```diff
@@ -42,7 +42,9 @@ export type ReactYield = {
   continuation: mixed
 };
 
-exports.createCoroutine = function<T>(children : mixed, handler : CoroutineHandler<T>, props : T, key : ?string = null) : ReactCoroutine {
+exports.createCoroutine = function<T>(
```

Comment:
GitHub supports ["squash and merge"](https://github.com/blog/2141-squash-your-commits) now so feel free to add a new commit.